EIPS/eip-1.md

```diff
@@ -212,6 +212,24 @@ Permitted Consensus Layer Specifications URLs must anchor to a specific commit,
 ^https://github.com/ethereum/consensus-specs/blob/[0-9a-f]{40}/.*$
 ```
 
+### Networking Specifications
+
+Links to the Ethereum Networking Specifications may be included using normal markdown syntax, such as:
```

Comment:
```suggestion
Links to specific commits and files of specific commits of the Ethereum Networking Specifications GitHub repository may be included using normal markdown syntax, such as:
```

Reply:
The text should be consistent between consensus and networking specs.

Reply:
Can I suggest that the text be changed for both, then?